src/test/java/htsjdk/samtools/cram/io/ExternalCompressionTest.java

```diff
@@ -0,0 +1,54 @@
+package htsjdk.samtools.cram.io;
+
+import htsjdk.HtsjdkTest;
+import htsjdk.samtools.cram.encoding.rans.RANS;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class ExternalCompressionTest extends HtsjdkTest {
+    public static final File BZIP2_FILE = new File("src/test/resources/htsjdk/samtools/cram/io/bzip2-test.bz2");
+    public static final byte [] TEST_BYTES = "This is a simple string to test compression".getBytes();
```

Comment:
odd spacing, I would expect to not have a space before `[]`
```suggestion
    public static final byte[] TEST_BYTES = "This is a simple string to test compression".getBytes();
```